index.bs

```diff
@@ -905,6 +905,49 @@ authorizing an authenticator with which to complete the operation.
 </div>
 
 
+
+### Inform user platform authenticator capability - isPlatformAuthenticatorReady() method ### {#isPlatformAuthenticatorReady}
```

Comment:
Maybe: "Advertise WebAuthn Support to Users - isPlatformAuthenticatorReady() method"?